test/acceptance/support/IntegrationPatience.scala

```diff
@@ -0,0 +1,13 @@
+package acceptance.support
+
+import org.scalatest.concurrent.Eventually.PatienceConfig
+import org.scalatest.concurrent.Futures.scaled
+import org.scalatest.time.{Seconds, Span}
+
+trait IntegrationPatience
+
+object IntegrationPatience {
```

Comment:
Hi @kyle-bowden, just having a look at this - does there need to be the `object IntegrationPatience`, or can this be just the trait with the `val` defined?

Reply:
Hi @JoPintoPaul 
I tried that before but I had a weird import issue, I have since fixed that and updated the code.